src/tests/objects.rs

```diff
@@ -115,17 +115,19 @@ fn test_create_objects() {
         &bool_values,
     ));
 
-    let _ = ret_or_panic!(import_object(
-        session,
-        CKO_CERTIFICATE,
-        &[(CKA_CERTIFICATE_TYPE, CKC_X_509)],
-        &[
-            (CKA_CHECK_VALUE, "ignored".as_bytes()),
-            (CKA_SUBJECT, "subject".as_bytes()),
-            (CKA_VALUE, "value".as_bytes())
-        ],
-        &[(CKA_TOKEN, true), (CKA_TRUSTED, false)],
-    ));
+    if testtokn.dbtype != "nssdb" {
```

Comment:
Probably misread. Sorry. It should have been on the DATA import if I see right.

Reply:
Ok, this was just in combination with FIPS mode for some reason
```
thread 'tests::objects::test_create_objects' panicked at src/tests/objects.rs:123:13:
```
I will have to dig into this deeper to figure out why it fails only in FIPS + NSSDB

Reply:
Ok, I tried to debug this with gdb and I have no idea why the combination of NSSDB and FIPS makes this panic (unless some weird optimization is going on there).

If I see right, we never reach the `fn_create_object()`, but the `import_object()` test helper does not do anything suspicious related to NSSDB nor FIPS.

Reply:
Do you get back a specific error number?
I wonder if the pbkdf2 used to generate the key to encrypt the data fails because the salt is to short or something like that.

Reply:
Not sure, if the 32 is the specific error:
```
thread 'tests::objects::test_create_objects' panicked at src/tests/objects.rs:123:13:
32
```
https://github.com/latchset/kryoptic/actions/runs/12236641232/job/34130555468?pr=122

That would map back to `CKR_DATA_INVALID`. Are there some parts of the certificates encrypted at all?